src/main/kotlin/device/application/DeviceRemover.kt

```diff
@@ -0,0 +1,14 @@
+package device.application
+
+import device.domain.DeviceRepository
+import device.domain.exceptions.NonExistentDeviceException
+import shared.domain.Identifier
+
+class DeviceRemover(
+    private val repository: DeviceRepository,
+) {
+    fun remove(deviceId: String) = remove(Identifier(deviceId))
```

Comment:
Documentation is missing...

![](https://media.giphy.com/media/4Zgy9QqzWU8C3ugvCa/giphy.gif)

Reply:
![](https://media.giphy.com/media/WJmgoToA78oyaUi7GL/giphy.gif)